Modules/hpe3par_snapshot.py

```diff
@@ -435,13 +447,134 @@ def restore_snapshot_online(
         snapshot_name,
         {})
 
+def create_schedule(
+        client_obj,
+        storage_system_ip,        
+        storage_system_username,
+        storage_system_password,
+        schedule_name,
+        snapshot_name,
+        base_volume_name,
+        read_only,
+        expiration_time,
+        retention_time,
+        expiration_unit,
+        retention_unit,
+        task_freq, task_freq_custom):
+    if storage_system_username is None or storage_system_password is None:
+        return (
+            False,
+            False,
+            "Schedule creation failed. Storage system username or password is \
+null",
+            {})
+    if schedule_name is None:
+        return (
+            False,
+            False,
+            "Schedule create failed. Schedule name is null",
+            {})                 
+    if snapshot_name is None:
```

Comment:
Again, what is the pupose of this check if we are not using snapshot name variable anywhere

Reply:
You can keep the timestamp, but append to the volume name. We can then get rid of snapshot name